Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -1,697 +1,92 @@
-  list-style: none;
+  position: relative;
 }
 
-/* Applied to the node's children area if the nodes should not be connected by lines. */
-ul.treeViewNodeChildrenNoLines
+ul.treeViewRoot,
+ul.treeViewRoot ul.treeViewNodeChildren,
+ul.treeViewRoot ul.treeViewNodeChildrenNoLines,
+ul.treeViewRoot ul.treeViewTopLevelNodeChildren
 {
-  margin: 0;
   padding: 0;
   list-style: none;
 }
 
-ul.treeViewRoot li
+ul.treeViewRoot span.treeViewNodeHeadSelected a:focus,
+ul.treeViewRoot li[role=treeitem]
 {
   outline: none;
 }
 
-ul.treeViewRoot > li,
-ul.treeViewTopLevelNodeChildren > li
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHead a > span
 {
+  text-decoration: underline;
 }
 
-ul.treeViewNodeChildren > li,
-ul.treeViewNodeChildrenNoLines > li
+ul.treeViewRoot span.treeViewNode:hover .treeViewNodeHead:before,
+ul.treeViewRoot span.treeViewNode .treeViewNodeHeadSelected:before,
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHead:before,
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHeadSelected:before
 {
+  background-color: var(--color-highlight);
 }
 
-/* Applied to the node itself */
-span.treeViewNode
+ul.treeViewRoot .treeViewNodeHead:before,
+ul.treeViewRoot .treeViewNodeHeadSelected:before
 {
-  cursor: default;
-  display: inline-block;
+  content: " ";
+  position: absolute;
+  top: 0;
+  left: 0;
   width: 100%;
-  box-sizing: border-box;
-}
-
-/* We want the tree node to span the entire line to better handle clicking and styling of the selected line.
-  Since it is not possible to use a nesting-count in CSS, we need to do an explicit nesting for a reasonable depth (50 levels). 
-  Once the maximum depth is reached, the formatting and behavior will revert to nesting via parent padding. */
-ul.treeViewRoot span.treeViewNode
-{
-  padding-left: calc(19px * 0);
-}
-
-ul.treeViewRoot ul span.treeViewNode
-{
-  padding-left: calc(19px * 1);
-}
-
-ul.treeViewRoot ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 2);
-}
-
-ul.treeViewRoot ul ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 3);
-}
-
-ul.treeViewRoot ul ul ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 4);
-}
-
-ul.treeViewRoot ul ul ul ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 5);
-}
-
-ul.treeViewRoot ul ul ul ul ul  ul span.treeViewNode
-{
-  padding-left: calc(19px * 6);
-}
-
-ul.treeViewRoot ul ul ul ul ul  ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 7);
-}
-
-ul.treeViewRoot ul ul ul ul ul  ul ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 8);
-}
-
-ul.treeViewRoot ul ul ul ul ul  ul ul ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 9);
-}
-
-ul.treeViewRoot ul ul ul ul ul  ul ul ul ul ul span.treeViewNode
-{
-  padding-left: calc(19px * 10);
+  bottom: 0;
+  z-index: -1;
 }
 
-ul.treeViewRoot ul ul ul ul ul  ul ul ul ul ul  ul span.treeViewNode
+ul.treeViewRoot .treeViewNode,
+ul.treeViewRoot .treeViewNodeHead > a,
+ul.treeViewRoot .treeViewNodeHeadSelected > a
```

Comment:
Please make this consistent. You're using `span.treeViewNode` in most of the rest of the file.